### PR TITLE
Reduce build time

### PR DESCRIPTION
**Pull request by jreidinger**
this change removes yast2-mail from critical path of minimal rebuild time.

### CODE REVIEW

File: package/yast2-mail.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Sep 20 07:56:27 UTC 2016 - jreidinger@suse.com
+
+- Reduce build dependencies to speed up build (bnc#999203)
```
Comment (lslezak) [inline package/yast2-mail.changes]: NP: I prefer SUSE bugzilla instead of Novell, i.e. `bsc#...`

File: package/yast2-mail.spec [hunk 2]
```diff
@@ -17,46 +17,33 @@
...
-Obsoletes:      yast2-mail-server <= %{version}
-Provides:       yast2-mail-plugins = %{version}
-Obsoletes:      yast2-mail-plugins <= %{version}
-Conflicts:	aaa_base < 10.3
-BuildArch:	noarch
+Requires:       yast2 >= 3.1.179
+Requires:       yast2-ldap
+Requires:       yast2-users
+Provides:       yast2-config-network:/usr/lib/YaST2/clients/lan_sendmail.ycp
+Conflicts:      aaa_base < 10.3
```
Comment (lslezak) [inline package/yast2-mail.spec]: NP: `10.3` seems to be really old, I'd drop this as well...

File: package/yast2-mail.spec [hunk 3]
```diff
@@ -17,46 +17,33 @@
...
+BuildArch:      noarch
 Requires:       yast2-ruby-bindings >= 1.0.0
 
-Summary:	YaST2 - Mail Configuration
+Summary:        YaST2 - Mail Configuration
+License:        GPL-2.0+
```
Comment (lslezak) [inline package/yast2-mail.spec]: We use `GPL-2.0` (without +) in the other packages, should we change it here as well?

File: package/yast2-mail.spec [hunk 4]
```diff
@@ -17,46 +17,33 @@
...
-Provides:       yast2-mail-server = %{version}
-Obsoletes:      yast2-mail-server <= %{version}
-Provides:       yast2-mail-plugins = %{version}
-Obsoletes:      yast2-mail-plugins <= %{version}
-Conflicts:	aaa_base < 10.3
-BuildArch:	noarch
+Requires:       yast2 >= 3.1.179
+Requires:       yast2-ldap
+Requires:       yast2-users
+Provides:       yast2-config-network:/usr/lib/YaST2/clients/lan_sendmail.ycp
```
Comment (lslezak) [inline package/yast2-mail.spec]: I guess this is not needed anymore, moreover this YCP file has been dropped...

File: package/yast2-mail.spec (hunk 3 above)
Comment (jreidinger) [inline package/yast2-mail.spec]: I am not sure if I can freely change license. I know from past that some newer yast modules have different license

Comment (mvidner) [inline package/yast2-mail.spec]: If it ain't broken don't fix it.

Comment (jsrain) [inline package/yast2-mail.spec]: Don't change the license, there might be code of 3rd parties in YaST (patches from community or partners etc.).
The rule was not to use GPL2+ for new packages, those should get 2-only.

**Pull request merged**